knowledge-base/how-to-convert-svg-to-png.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Covert SVG graphics to PNG picture
+description: This article demonstrates how to display SVG graphics in PictureBox by converting it to PNG
+type: how-to
+page_title: How to convert SVG to PNG 
+slug: how-to-convert-svg-to-png
+position: 
+tags: 
+ticketid: 1400737
+res_type: kb
+---
+
+## Environment
+<table>
+	<tr>
+		<td>Product Version</td>
+		<td>13.0.19.222</td>
+	</tr>
+	<tr>
+		<td>Product</td>
+		<td>Progress® Telerik® Reporting</td>
+	</tr>
+</table>
+
+
+## Description
+Currently, the [PictureBox](../report-items-picture-box) can hold only the formats supported by GDI+ (BMP, GIF, JPEG, EXIF, PNG, and TIFF). This KB article demonstrates how to convert SVG graphics to PNG picture through the examples below. Note that a  third party package- Svg should be added to the with NuGet packages.  
+
+## Solution
+The first option is by using [UserFunction](../expressions-user-functions) when the report is developed from the Standalone designer. It can be implemented as the code snippet below:
+```
+public static Bitmap ConvertToBitmap(string url)
+{
+    var svgDocument = Svg.SvgDocument.Open(url);
+    svgDocument.ShapeRendering = SvgShapeRendering.Auto;
+ 
+    Bitmap bmp = svgDocument.Draw(120, 120); // Draw Bitmap in any Size you need - for example 120px x 120px
+    return bmp;
+}
+```
```

Comment:
You may use syntax like 
```CSharp _or_ ```XML
to specify the programming language and benefit from auto-formatting.